samza-core/src/test/java/org/apache/samza/zk/TestZkLeaderElector.java

```diff
@@ -409,12 +409,12 @@ public void testAmILeader() {
     // Processor-1
 
     ZkUtils zkUtils1 = getZkUtilsWithNewClient();
-    ZkLeaderElector leaderElector1 = new ZkLeaderElector("1", zkUtils1, null);
+    ZkLeaderElector leaderElector1 = new ZkLeaderElector("1", zkUtils1);
```

Comment:
Uses a new constructor based on the change which includes validation of paths on zkutils which wasn't done previously. 

Is that a concern for tests in terms of mocking additional behavior?